tutorials/plugins/running_code_in_the_editor.rst

```diff
@@ -105,18 +105,20 @@ Here is how a ``_process()`` function might look for you:
         // Code to execute both in editor and in game.
     }
 
-.. note::
+Important information
+---------------------
 
-    Modifications in the editor are permanent. For example, in the following
-    case, when we remove the script, the node will keep its rotation. Be careful
-    to avoid making unwanted modifications.
+Any other GDScript that your tool script uses must *also* be a tool. Any
+GDScript without ``@tool`` imported into the editor will act like an empty file!
```

Comment:
Do we "import" scripts? Is this terminology used elsewhere? I don't think it makes sense to use it, imports are files that are modified and converted no?

Reply:
That terminology comes from this note on the making plugins page, which I modified a little for this PR

"In addition to the EditorPlugin script, any other GDScript that your plugin uses must also be a tool. Any GDScript without @tool imported into the editor will act like an empty file!"

Reply:
That should probably be modified separately later, but I don't think using "imported" here as it's not correct for scripts